jgrapht-core/src/main/java/org/jgrapht/alg/shortestpath/DeltaSteppingShortestPath.java

```diff
@@ -63,7 +63,10 @@
  * </ul>
  *
  * <p>
- * For parallelization, this implementation relies on the {@link ExecutorService}.
+ * For parallelization, this implementation relies on the {@link ThreadPoolExecutor}
+ * which is supplied to this algorithm from outside. This algorithm does not manages the
```

Comment:
"does not manages" -> "does not manage"

same typo copied elsewhere in this PR